docs/includes/db_connection_security.rst

```diff
@@ -0,0 +1,12 @@
+Database Connection Security
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the connector configuration you will notice there are no security parameters. This is because
+SSL is not part of the JDBC standard and will depend on the JDBC driver in use. In general, you
+will need to configure SSL via the ``connection.url`` parameter. For example, with MySQL it would
+look like:
+
+  ..sourcecode properties:
```

Comment:
this should be `.. sourceode:: properties`. right now it is rendering weirdly because it's not being interpreted as an rst command

Reply:
yeah, the output is messed up 
![image](https://user-images.githubusercontent.com/11722533/34504220-07247b8a-efd2-11e7-8515-6648d0a8d2db.png)